src/hypertable.c

```diff
@@ -2461,14 +2461,17 @@ ts_chunk_get_osm_slice_and_lock(int32 osm_chunk_id, int32 time_dim_id, LockTuple
 				.lockmode = tuplockmode,
 				.waitpolicy = LockWaitBlock,
 			};
+			ScanTupLock *const tuplock_ptr = RecoveryInProgress() ? NULL : &tuplock;
```

Comment:
Could you please add a comment to the one (similar to this one: https://github.com/timescale/timescaledb/commit/23cbb0998300ecf85e6d16bf4d29c3fe8f6bdd8b)  here?
Otherwise, its not clear why this is needed.

Reply:
Thanks! I added a comment.